packages/orbit-components/src/Timeline/TimelineStep/primitives/StyledIconWrapper.js

```diff
@@ -4,12 +4,11 @@ import styled from "styled-components";
 import defaultTheme from "../../../defaultTheme";
 
 const StyledIconWrapper = styled.div`
-  display: flex;
-  align-items: center;
-  justify-content: center;
   min-width: ${({ theme, mobile }) => mobile && theme.orbit.spaceLarge};
   min-height: ${({ theme }) => theme.orbit.spaceLarge};
   z-index: 1;
+  text-align: center;
+  line-height: 1.3;
```

Comment:
Could you just explain why value `1.3` exactly? 🙂

Reply:
@vepor 

well, it looked like optimal line-height in IE and Chrome, we have tokens for 1.2 and 1.4 🤔

Reply:
Not sure if we can use `flexbox` as we do for now. It would make sense to use height token then. If not I'm fine with explicit `line-height` also. Usage of some specific token would be the best, if that won't work, please just attach some comment 😉